tests/unit/amo/components/TestRatingsByStar.js

```diff
@@ -123,7 +130,9 @@ describe(__filename, () => {
 
   it('renders a loading state without an add-on', () => {
     const root = render({ addon: null });
+    const tableRow = root.find('.RatingByStar-table-row');
 
+    expect(tableRow.at(0)).not.toHaveProp('onClick');
```

Comment:
Thanks for adding this. We should also have an assert that checks that the prop `title` also does not exist.

Reply:
Ohk! I will add that.